specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -385,35 +391,35 @@
       }
     },
     "ContainerState": {
-      "description": "The container state.",
+      "description": "The container instance state.",
       "type": "object",
       "properties": {
         "state": {
           "type": "string",
-          "description": "The state of this container"
+          "description": "The state of the container instance."
         },
         "startTime": {
           "type": "string",
           "format": "date-time",
-          "description": "Date/time when the container state started."
+          "description": "The date-time when the container instance state started."
         },
         "exitCode": {
           "type": "integer",
-          "description": "The container exit code."
+          "description": "The container instance exit code. Exit codes map to output from `docker run`."
```

Comment:
How about:

"Exit codes correspond to those of the `docker run` command."

It feels a bit odd that that we're referencing an external tool in our own API reference documentation, but in this case, if it provides the best and most succinct explanation, it might be appropriate. Not sure there's another way to better communicate this, but wanted to call it out as a potential gotcha we might hit later on.

Reply:
@yolo3301 is there another way to reference the exit codes that can abstract us from docker?